ViewGeneratorCore/build/ViewGeneratorCore.targets

```diff
@@ -38,13 +38,16 @@
     </PropertyGroup>
   </Target>
 
-  <ItemGroup>
-    <VGC_Inputs Include="$(OverrideProjectDir)**\*.tsx" Exclude="$(OverrideProjectDir)node_modules\**\*.tsx" />
-    <VGC_Inputs Include="$(OverrideProjectDir)**\*.ts" Exclude="$(OverrideProjectDir)node_modules\**\*.ts" />
-    <UpToDateCheckInput Include="@(VGC_Inputs)" Exclude="@(UpToDateCheckInput)" />
-  </ItemGroup>
-
-  <Target Name="VGC_GenerateView" Inputs="@(VGC_Inputs)" Outputs="VGC_Build.cache" Condition="'$(UsingMicrosoftNETSdk)' != 'true' Or '$(DesignTimeBuild)' != 'true'" BeforeTargets="CustomTransformDuringBuild;TransformDuringBuild;AfterResolveReferences" DependsOnTargets="VGC_SetVariables;VGC_Setup">
+    <Target Name="VGC_GetInputs" DependsOnTargets="VGC_SetVariables;VGC_Setup">
+        <Exec ConsoleToMSBuild="True" Command="&quot;$(ViewGeneratorCoreNodeJsExe)&quot; &quot;$(MSBuildThisFileDirectory)..\tools\node_modules\@outsystems\ts2lang\ts2lang-main.js&quot; -l -f &quot;$(ProjectDir)ts2lang.json&quot; -t &quot;$(MSBuildThisFileDirectory)..\tools\ViewGenerator.js&quot;">
```

Comment:
\ not unix compliant

Reply:
This is very strange. This is working on ts2lang without those slashes, msbuild seems to be converting it.

Look at this test I implemented.
![image](https://user-images.githubusercontent.com/74782571/109357986-f82d6580-787a-11eb-9b75-363d32a16bd1.png)

Reply:
@fernandomarquesgarcia do you know which targets need the \ properly set?

Reply:
hi, any update on this?

Reply:
Hi,
From what I could come up with by testing with a simple console project having one target that prints a message:
-  <Message Importance="high" Text="c:\ola.txt"/> --> **Prints "c:\ola.txt"**
- <Message Importance="high" Text="$(OutDir)c:\ola.txt" /> --> **Prints bin/Debug/net5.0/c:/ola.txt**
-    <Target Name="SetMyVars">
        <PropertyGroup>
            <MyTestPath>c:\ole.txt</MyTestPath>
        </PropertyGroup>
    </Target>
    <Target Name="Test" AfterTargets="BeforeBuild" DependsOnTargets="SetMyVars">
        <Message Importance="high" Text="$(MyTestPath)c:\ola.txt" />
    </Target>
This prints **c:\ole.txtc:\ola.txt**
-    <Target Name="SetMyVars">
        <PropertyGroup>
            <MyTestPath>$(OutDir)c:\ole.txt</MyTestPath>
        </PropertyGroup>
    </Target>
    <Target Name="Test" AfterTargets="BeforeBuild" DependsOnTargets="SetMyVars">
        <Message Importance="high" Text="$(MyTestPath)c:\ola.txt" />
    </Target>
THIS Prints **bin/Debug/net5.0/c:/ole.txtc:/ola.txt**

On file handling system targets like Copy or Include, no matter what path separator we use, msbuild translates it, so this exemple works:
**<Copy SourceFiles="test//\\\Program2.cs" DestinationFiles="test\\\\///\Program5.cs"></Copy>**

**So Summing up:**
-Attibutes that are supposed to have paths (like Source Files on Copy or Include) always translate the paths according to host system.
-Other atributes (like Text on Message) will translate the path separator if used in conjunction with a MSBUild Macro that returns a path (On Mac concatenating a path with $(ProjectDir) translates the path but doing it with $(FxCopDir) doesn't work because the macro doesn't return a value )

My Advise:
Always use slash (/) because it works on both platforms although a backslash will be translated in innumerous cases.

Reply:
thanks, changed!